### DISCUSSION TITLE
very high confidence interval of parameter estimates

### DISCUSSION THREAD

**Discussion by hainiwen** [Open-Systems-Pharmacology/Forum#1643] [Questions & Problems]
Hi there,
I get this very big confidence interval for the parameter estimates. I wonder what does that say about my model. Also, I'm a bit confused about how did Mobi manage to calculate a confidence interval when everything is so far individual (mean data with error bar)
![Capture](https://github.com/Open-Systems-Pharmacology/Forum/assets/56790121/334a62ba-24ac-46df-87b8-cccbe62b77ce)

Lots of thanks in advance!

**Comment by hainiwen** [comment]
Also, what is the difference between confidence interval, visual predictive check interval, prediction interval, that plots for parameter estimation?

**Comment by wilbertdew** [comment]
hi @hainiwen these confidence intervals are calculated based on the uncertainty of your parameter estimation, i.e. how much does the total error change if you change the estimated parameter value. The very large intervals indicate your data is not so informative to estimate these parameters precisely. The confidence interval graph uses these uncertainties in the parameter values to calculated how this translates into uncertainties of predicted output values of your model. The VPC calculates the standard deviation of the distribution of the residuals between observed data and model predictions, and uses that SD to calculate an interval around the predicted output values, per definition this interval covers 95% of the observed data, so that does not inform you on the goodness of fit, but you could use it to identify datapoints that are far out of the predicted interval. the prediction interval, finally, adds these two variabilities to predict the range in which you would expect future data to appear.

**Reply by hainiwen** [reply]
thanks!! I'll also check the reference in the manual!
